src/main/java/CalculatorInputStringParser.java

```diff
@@ -0,0 +1,29 @@
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class CalculatorInputStringParser {
+	private static final Pattern PATTERN_DEFAULT_DELIMITERS = Pattern.compile("[,:]");
+	private static final Pattern PATTERN_FOR_CAPTURE_CUSTOM_DELIMITER_AND_NUMBERS = Pattern.compile("//(.)\n(.*)");
+
+	public static List<Integer> parse(String inputString) {
+		Matcher matcher = PATTERN_FOR_CAPTURE_CUSTOM_DELIMITER_AND_NUMBERS.matcher(inputString);
+		if (matcher.find()) {
+			return getIntegersWhenHasCustomDelimiter(matcher);
+		}
+		return getIntegers(inputString);
+	}
+
+	private static List<Integer> getIntegers(String inputString) {
+		String[] tokens = PATTERN_DEFAULT_DELIMITERS.split(inputString);
+		return Arrays.stream(tokens).map(Integer::parseInt).collect(Collectors.toList());
```

Comment:
```suggestion
		return Arrays.stream(tokens)
			.map(Integer::parseInt)
			.collect(Collectors.toList());
```

라인 단위로 구분한다면 가독성이 상승할 것 같아요!